Shared/Samples/Set map image layer sublayer visibility/SetMapImageLayerSublayerVisibilityView.swift

```diff
@@ -0,0 +1,104 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct SetMapImageLayerSublayerVisibilityView: View {
```

Comment:
Should this be renamed to just sublayer visibility?
```suggestion
struct SetMapImageSublayerVisibilityView: View {
```

Reply:
Let's keep consistent with the design title.

Reply:
sounds good 🧐